app/assets/javascripts/discourse/tests/acceptance/sidebar-narrow-desktop-test.js

```diff
@@ -0,0 +1,54 @@
+import { test } from "qunit";
+
+import { click, visit } from "@ember/test-helpers";
+
+import { acceptance, exists } from "discourse/tests/helpers/qunit-helpers";
+
+acceptance("Sidebar - Narrow Desktop", function (needs) {
+  needs.user();
+
+  needs.settings({
+    enable_experimental_sidebar_hamburger: true,
+    enable_sidebar: true,
+  });
+
+  needs.narrowDesktopView();
```

Comment:
Instead of forcing the narrow desktop view straight away, what do you think about faking the transition to a narrow width state instead by setting narrow desktop view on site after we've rendered the main sidebar?